wlauto/result_processors/cpustate.py

```diff
@@ -163,8 +163,10 @@ def process_iteration_result(self, result, context):
         self.logger.debug('Generating power state reports from trace...')
         if self.create_timeline:
             timeline_csv_file = os.path.join(context.output_directory, 'power_states.csv')
+            cpu_utilisation = os.path.join(context.output_directory, 'cpu_utilisation.csv')
```

Comment:
This should have its own parameter with a proper description instead of being a part of create_timeline.